yt_dlp/extractor/goplay.py

```diff
@@ -0,0 +1,400 @@
+import base64
+import binascii
+import datetime
+import hashlib
+import hmac
+import json
+import os
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class GoPlayIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.)?goplay\.be/video/((?P<series>[^/]+)/(?P<season>[^/]+)/|)(?P<episode>[^/]+)'
+
+    _NETRC_MACHINE = 'goplay'
+
+    _TESTS = [{
+        'url': 'https://www.goplay.be/video/de-container-cup/de-container-cup-s3/de-container-cup-s3-aflevering-2#autoplay',
+        'info_dict': {
+            'id': '9c4214b8-e55d-4e4b-a446-f015f6c6f811',
+            'ext': 'mp4',
+            'title': 'S3 - Aflevering 2',
+            'series': 'De Container Cup',
+            'season': 'Season 3',
+            'season_number': 3,
+            'episode': 'Episode 2',
+            'episode_number': 2,
+        },
+        'skip': 'This video is only available for registered users'
+    }, {
+        'url': 'https://www.goplay.be/video/a-family-for-thr-holidays-s1-aflevering-1#autoplay',
+        'info_dict': {
+            'id': '74e3ed07-748c-49e4-85a0-393a93337dbf',
+            'ext': 'mp4',
+            'title': 'A Family for the Holidays',
+        },
+        'skip': 'This video is only available for registered users'
+    }]
+
+    def _initialize_pre_login(self):
+        self._id_token = None
+
+    def _perform_login(self, username, password):
+        self.report_login()
+        aws = AwsIdp(ie=self, pool_id='eu-west-1_dViSsKM5Y', client_id='6s1h851s8uplco5h6mqh1jac8m')
+        self._id_token, _ = aws.authenticate(username=username, password=password)
+
+    def _real_initialize(self):
+        if not self._id_token:
+            raise self.raise_login_required(method='password')
+
+    def _real_extract(self, url):
+        display_id = url.split('#')[0].split('/')[-1]
+        webpage = self._download_webpage(url, display_id)
+        video_data_json = self._html_search_regex(r'<div\s+data-hero="([^"]+)"', webpage, 'video_data')
+        video_data = self._parse_json(video_data_json.replace('quot;', '"'), display_id)['data']
+
+        movie = video_data['movie']
+        if movie:
+            video_id = movie['videoUuid']
+            data = {
```

Comment:
```suggestion
            info_dict = {
```
That is the conventionally used name